docs/build/wasp-evm/v1.0.0-rc.6/docs/how-tos/integrate-your-exchange.mdx

```diff
@@ -0,0 +1,164 @@
+
+# ShimmerEVM Exchange Integration
+
+If you already support EVM, then you probably are already familiar with what you'll need to integrate ShimmerEVM. As
+with most networks, you should be able to use the same API as [go-ethereum](https://geth.ethereum.org/) taking into account the
+[compatibility and limitations](../getting-started/compatibility.md), use the ShimmerEVM RPC
+URL (https://json-rpc.evm.shimmer.network) and ChainId (<ChainId url='https://json-rpc.evm.shimmer.network'/>), and run
+our [Hornet](/hornet/welcome/) node with the [Wasp](/wasp/running-a-node/) INX-plugin.
+
+## ShimmerEVM Network Information
+
+<AddToMetaMaskButton cfg={EVMNetworks['shimmerevm']} />
+
+| Base Token          | Protocol  | Chain ID                                              | RPC URL                                                                       | Explorer                             |
+|---------------------|-----------|-------------------------------------------------------|-------------------------------------------------------------------------------|--------------------------------------|
+| Shimmer Token (SMR) | ISC / EVM | <ChainId url='https://json-rpc.evm.shimmer.network'/> | https://json-rpc.evm.shimmer.network or wss://ws.json-rpc.evm.shimmer.network | https://explorer.evm.shimmer.network |
+
+#### Additional Info
+
+| Chain Address                                                                                                                                                                    | Alias ID                                                           |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| [smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s](https://explorer.shimmer.network/shimmer/addr/smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s) | 0xccc7018e4fa63e5014332f45ddc8a5450da89572676d12d4d5e51c98d64155b3 |
+
+## Running a Hornet Node with Wasp
+
+### Install Hornet
+
+The easiest way to get your [Hornet](/hornet/welcome/) node up and running is using the
+provided [Docker image](/hornet/how_tos/using_docker/) which already includes the [Wasp](/wasp/running-a-node/)
+INX-plugin.
+
+### Post-Install Tasks
+
+After you have [installed your node](#install-hornet), you should make sure you:
+
+* [Configure Hornet's HTTP REST API](/hornet/how_tos/post_installation/#configuring-http-rest-api) to operate securely.
+* Review the configuration options for both
+[Hornet](/hornet/references/configuration/) and [Wasp](/wasp/configuration/).
+* [Add peers to your node](/hornet/references/peering/).
+
+### Interact with the ShimmerEVM Chain
+
+You can use [`wasp-cli`](/wasp-cli/how-tos/wasp-cli/) to safely interact with
+and [manage](wasp-cli/how-tos/chain-management/) your Wasp instance.
+
```

Comment:
That's Hornet. This guide is for ShimmerEVM specifically no? So if you choose to run your own Wasp node to connect to, instead of the public endpoint, you need to get peered with other Wasp nodes in the ShimmerEVM network. But afaik that's a permissioned group currently, so you need to request access somewhere right?